L3-L4-OOP/L4-Zad.3-AreaAndPerimeter/src/main/java/pl/babiarz/Circle.java

```diff
@@ -0,0 +1,21 @@
+package pl.babiarz;
+
+public class Circle implements AreaAndPerimeter {
+
+    private double r;
+    private static final double pi = 3.14;
```

Comment:
stałe piszemy TROCHE_INACZEJ :)  https://github.com/mtszpater/future-collars-java-intro-faq#sta%C5%82e

Reply:
z resztą inny hint - takie matematyczne stałe są już w klasie Math np. Math.PI  https://codegym.cc/groups/posts/575-mathpi-in-java

Reply:
Zrobiłem z Math.PI, ale chyba nie kumam co było źle w kontekście tego pierwszego komentarza, bo wydaje mi się, że było tak jak w tym faqu. Sama nazwa zmiennej była zła? Mógłbyś mi napisać jak to ma wyglądać? Nawiasem mówiąc napisałem ją dokładnie tak jak zasugerował mi drugi Mateusz w komentarzu do poprzedniego zadania, w którym też było pole i obwód koła :)